staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1017,6 +1017,15 @@ message Device {
     string permissions = 3;
 }
 
+// CDIDevice specifies a CDI device information.
+message CDIDevice {
```

Comment:
@klueska @dims @SergeyKanzhelev @thockin @smarterclayton @derekwaynecarr @dchen1107 
CRI is a v1 API and DRA is an alpha API, what are the stability expectactions of this new field added here?
Can this field be removed or modified?